Sources/MarkCodable/Decoder/MarkKeyedDecoding.swift

```diff
@@ -80,7 +80,8 @@ struct MarkKeyedDecoding<Key: CodingKey>: KeyedDecodingContainerProtocol {
     }
 
     private func unbox<T: Decodable & StringInitializable>(_ key: Key) throws -> T {
-        guard let value = data[key.stringValue] else {
+        let nestedPath = codingPath + [key]
```

Comment:
This makes very much sense — it was a big oversight we didn't have tests to catch this, thanks for the fix